src/bridge.ts

```diff
@@ -686,6 +693,16 @@ export class Bridge {
         this.setupIntentCulling();
 
         await this.loadDatabases();
+
+        this.userActivityTracker = new UserActivityTracker(
+            this.opts.userActivityTrackerConfig || UserActivityTrackerConfig.DEFAULT,
+            await this.botSdkAS.botClient.getSafeAccountData(BRIDGE_USER_ACTIVITY_STORAGE_KEY, { users: {} }),
```

Comment:
Like you said, I'd be a little worried about storing this much data in account data (despite what Teams does). Perhaps we could make this an interface for bridge developers to implement?